forms/NullableField.php

```diff
@@ -77,53 +100,80 @@ public function getIsNullId() {
 	}
 
 	/**
-	 * (non-PHPdoc)
-	 * @see framework/forms/FormField#Field()
+	 * @param array $properties
+	 *
+	 * @return string
 	 */
 	public function Field($properties = array()) {
-		if ( $this->isReadonly()) {
+		if($this->isReadonly()) {
 			$nullableCheckbox = new CheckboxField_Readonly($this->getIsNullId());
 		} else {
 			$nullableCheckbox = new CheckboxField($this->getIsNullId());
 		}
+
 		$nullableCheckbox->setValue(is_null($this->dataValue()));
 
-		return $this->valueField->Field() . ' ' . $nullableCheckbox->Field()
-			. '&nbsp;<span>' . $this->getIsNullLabel().'</span>';
```

Comment:
Could the layout be broken by removing the `&nbsp;`? Possibly this is intended to prevent the label wrapping to the next line?